Bool odom_node_rdy